app/models/merits_assessment.rb

```diff
@@ -1,23 +1,15 @@
 class MeritsAssessment < ApplicationRecord
   belongs_to :legal_aid_application
 
-  # defining before enum so it can be used in the enum
-  def self.prospect_likely_to_succeed
-    :likely
-  end
-
-  enum(
-    success_prospect: {
-      prospect_likely_to_succeed => '50% or better'.freeze,
-      marginal: 'Marginal 45 - 50%'.freeze,
-      borderline: 'borderline'.freeze,
-      uncertain: 'uncertain'.freeze,
-      poor: 'poor'.freeze
-    },
-    _prefix: true
-  )
+  enum success_prospect: {
+    likely: 'likely',
+    marginal: 'marginal',
+    poor: 'poor',
+    borderline: 'borderline',
+    not_known: 'not_known'
+  }, _prefix: :success_prospect
```

Comment:
I think these enum options should still be frozen. And `_prefix: true` does already set the prefix to match the enum name.